docs/reference/mapping/runtime.asciidoc

```diff
@@ -752,6 +761,79 @@ The API returns the following result. Without building your data structure in
 advance, you can search and explore your data in meaningful ways to experiment
 and determine which fields to index.
 
+Also, remember that `if` statement in the script?
+
+[source,txt]
+----
+if (clientip != null) emit(clientip);
+----
+
+If the script didn't include this condition, the result wouldn't include any
+documents. By including this condition, the script still returns documents if
+there's data that doesn't match the grok pattern.
+
+[source,console-result]
+----
+{
+  ...
+  "hits" : {
+    "total" : {
+      "value" : 1,
+      "relation" : "eq"
+    },
+    "max_score" : 1.0,
+    "hits" : [
+      {
+        "_index" : "my-index",
+        "_id" : "FdLqu3cBhqheMnFKd0gK",
+        "_score" : 1.0,
+        "_source" : {
+          "timestamp" : "2020-04-30T14:30:17-05:00",
+          "message" : "40.135.0.0 - - [30/Apr/2020:14:30:17 -0500] \"GET /images/hm_bg.jpg HTTP/1.0\" 200 24736"
+        },
+        "fields" : {
+          "http.clientip" : [
+            "40.135.0.0"
+          ],
+          "message" : [
+            "40.135.0.0 - - [30/Apr/2020:14:30:17 -0500] \"GET /images/hm_bg.jpg HTTP/1.0\" 200 24736"
+          ],
+          "timestamp" : [
+            "2020-04-30T19:30:17.000Z"
+          ]
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/\.\.\./"took" : $body.took,"timed_out" : $body.timed_out,"_shards" : $body._shards,/]
+// TESTRESPONSE[s/"_id" : "FdLqu3cBhqheMnFKd0gK"/"_id": $body.hits.hits.0._id/]
+
+[[runtime-examples-grok-range]]
+===== Search for documents in a specific range
+You can also run a <<query-dsl-range-query,range query>> that operates on the
+`timestamp` field. The following query returns any documents where the
+`timestamp` is greater than or equal to `2020-04-30T14:31:27-05:00`:
+
+[source,console]
+----
+GET my-index/_search
+{
+  "query": {
+    "range": {
+      "timestamp": {
+        "gte": "2020-04-30T14:31:27-05:00"
+      }
+    }
+  }
+}
+----
+// TEST[continued]
+
+The response includes the document where the log format doesn't match, but the
+timestamp falls within the defined range.
```

Comment:
I'm not sure what this bit adds to the doc. I think the timestamp isn't really *about* runtime fields. OTOH, I do think it'd be nice to have an example that uses a timestamp *and* the runtime field. This is how we hope people generally use runtime fields.

Reply:
@nik9000 , something like this?

`GET my-index/_search
{
  "query": {
    "range": {
      "timestamp": {
        "gte": "2020-04-30T14:31:27-05:00"
      }
    }
  }
  , "aggs": {
    "clientip": {
      "terms": {
        "field": "http.client.ip",
        "size": 10
      }
    }
  }
}`

Reply:
Yeah! That makes sense